treeherder/perf/auto_perf_sheriffing/bug_searcher.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from copy import deepcopy
+from datetime import datetime, timezone
+
+import requests
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class BugSearcher:
+    """Helper class to perform queries on Bugzilla.
+
+    TODO: Replace this with libmozdata Bugzilla class (potentially
+    replacing BugManager too)
+    """
+
+    def __init__(self):
+        self.bz_url = settings.BUGFILER_API_URL + "/rest/bug?"
```

Comment:
I would much rather use the format we do for line 9 of self.bz_url = settings.BUGFILER_API_URL + "/rest/bug" and then modify later to include the where clause